bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -8373,11 +8434,8 @@ public TourPerson getTourPerson() {
       return tourPhotos;
    }
 
-   /**
-    * @return Returns total recording time in seconds
-    */
-   public long getTourRecordingTime() {
-      return tourRecordingTime;
+   public long getTourRecordedTime() {
+      return tourRecordedTime;
```

Comment:
In some renaming actions, the method comment's got lost

Reply:
Fixed